draft-iyengar-quic-delayed-ack.md

```diff
@@ -291,19 +299,17 @@ to delaying of acknowledgments are reset.
 
 ## Response to Reordering {#reordering}
 
-As specified in Section 13.3.1 of {{QUIC-TRANSPORT}}, endpoints SHOULD send an
-acknowledgement immediately on receiving a reordered ack-eliciting packet,
-unless the peer has sent a `disable_ack_on_reordering` transport parameter,
-described below:
-
-disable_ack_on_reordering (0xXXXX):
+As specified in Section 13.3.1 of {{QUIC-TRANSPORT}}, endpoints are expected to
+send an acknowledgement immediately on receiving a reordered ack-eliciting
+packet. This extension modifies this behavior.
 
-: This optional transport parameter is sent by an endpoint that is reordering
-  tolerant or expects the connection to experience reordering. An endpoint that
-  receives this transport parameter MUST NOT make the exception of sending an
-  immediate acknowledgement when reordering is observed. This parameter is a
-  zero-length value, and is encoded as per Section 18 of {{QUIC-TRANSPORT}}.
+If the most recent ACK_FREQUENCY frame received from the peer has an `Ignore
+Order` value of `true` (0x01), the endpoint MUST NOT make the exception of
+sending an immediate acknowledgement when a packet is received out of order.
```

Comment:
How about ", the receiver MUST only send an acknowledgement in response to receiving Packet Tolerance packets or after the updated max_ack_delay has passed, and MUST NOT send an immediate acknowledgement in response to out of order packets."